src/Models/Income.cs

```diff
@@ -0,0 +1,12 @@
+namespace server.Models;
+
+public class Income
+{
+    public float Amount { get; set; }
```

Comment:
I decided to use the `decimal` type in the project, can elaborate on the choice if needed.

Reply:
What is the difference between them?